README.md

```diff
@@ -905,7 +906,7 @@ Retrieves all the names of the object's own enumerable properties.
 
 ### _.values
 
-Retrieve all the names of the object's own enumerable properties.
+Retrieve all the given object's own enumerable property values.
```

Comment:
we recently changed all verbs to third-person singular. So this should start with "Retrieve"